docs/user/04-metrics.md

```diff
@@ -19,41 +20,40 @@ While application logs and traces usually provide request-oriented data, metrics
 
 In the Telemetry module, a central in-cluster Deployment of an [OTel Collector](https://opentelemetry.io/docs/collector/) acts as a gateway. The gateway exposes endpoints for the [OpenTelemetry Protocol (OTLP)](https://opentelemetry.io/docs/specs/otlp/) for GRPC and HTTP-based communication using the dedicated `telemetry-otlp-metrics` service, to which all Kyma modules and users’ applications send the metrics data.
 
-Optionally, the Telemetry module provides a DaemonSet of an OTel Collector acting as an agent. This agent can pull metrics of a workload and the Istio sidecar in the [Prometheus pull-based format](https://prometheus.io/docs/instrumenting/exposition_formats) and can provide runtime-specific metrics for the workload.
+The feature is optional, if you don't want to use the Metrics feature, simply don't set up a MetricPipeline.
+
+Optionally, the Telemetry module provides a DaemonSet of an OTel Collector acting as an agent. This agent can retrieve metrics of a workload and the Istio sidecar in the [Prometheus pull-based format](https://prometheus.io/docs/instrumenting/exposition_formats) and can provide runtime-specific metrics for the workload.
 
 ![Architecture](./assets/metrics-arch.drawio.svg)
 
-1. An application (exposing metrics in OTLP) pushes metrics to the central metric gateway service.
-2. Activate the agent to scrape the metrics of an application (exposing metrics in Prometheus protocol) with an annotation-based configuration.
-3. Additionally, you can configure the agent to pull metrics of each Istio sidecar.
-4. The agent converts and pushes all collected metric data to the gateway in OTLP.
-5. The gateway discovers the metadata and enriches all received data with typical metadata of the source by communicating with the Kubernetes APIServer. Furthermore, it filters data according to the pipeline configuration.
-6. The `MetricPipeline` resource generates the config for the gateway, which specifies the target backend for the metric gateway.
-7. The backend can run within the cluster.
-8. If authentication has been set up, the backend can also run outside the cluster.
+1. An application (exposing metrics in OTLP) sends metrics to the central metric gateway service.
+2. An application (exposing metrics in Prometheus protocol) activates the agent to scrape the metrics with an annotation-based configuration.
+3. Additionally, you can activate the agent to pull metrics of each Istio sidecar.
+4. The agent supports collecting container metrics from the Kubelet and Kubernetes APIServer
```

Comment:
"collect" is a much simpler word than "retrieve", which we use in the diagram. I suggest to use "collect" in the diagram as well.

Reply:
done